secret_store/src/key_server_cluster/admin_sessions/share_add_session.rs

```diff
@@ -952,6 +950,8 @@ pub mod tests {
 				id: SessionId::default(),
 				self_node_id: NodeId::default(),
 				master_node_id: master_node_id,
+				configured_nodes_count: new_nodes_set.iter().chain(old_nodes_set.iter()).collect::<BTreeSet<_>>().len(),
```

Comment:
I tend to use `BTreeSet` across SS, because in some places (servers set change session, establishing connections in tests, serving contract requests, ...) nodes order is crucial. So - just trying to avoid exporting similar collection. Agree that here it isn't necessary - will update in a minute. Thanks!